### PR TITLE
EDU-13917: Update `product-card.mdx`

### PR DESCRIPTION
**Pull request by mariana-caetano**
#### Types of changes
- [ ] New content (guides, endpoints, app documentation)
- [x] Improvement (make a documentation even better)
- [ ] Fix (fix a documentation error)
- [ ] Spelling and grammar accuracy (self-explanatory)

### CODE REVIEW

**Review by hellofanny — APPROVED** [review]

File: docs/faststore/components/molecules/product-card.mdx [hunk 1]
```diff
@@ -719,6 +729,8 @@ Use the `ProductCard` to:
 
 `data-fs-product-card-taxes-label`
 
+`[data-fs-product-card-sponsored-label]`
```
Comment (hellofanny) [inline docs/faststore/components/molecules/product-card.mdx:732]:
```suggestion
`data-fs-product-card-sponsored-label`
```

Comment (hellofanny) [reply docs/faststore/components/molecules/product-card.mdx:732]: I think we don't use `[]` 😬 
image

Comment (Mariana-Caetano) [reply docs/faststore/components/molecules/product-card.mdx:732]: Thanks, Fanny!

File: docs/faststore/components/molecules/product-card.mdx [hunk 2]
```diff
@@ -719,6 +729,7 @@ Use the `ProductCard` to:
 
 `data-fs-product-card-taxes-label`
 
+`data-fs-product-card-sponsored-label`
```
Comment (mariana-caetano) [inline docs/faststore/components/molecules/product-card.mdx:732]:
```suggestion
`data-fs-product-card-sponsored-label`

```

**Review by carolinamenezes — APPROVED** [review]

**Pull request merged**